concepticondata/conceptlists/Gauchat-1925-480.tsv

```diff
@@ -0,0 +1,481 @@
+Gauchat-1925-480-73	73	Be careful	Prends garde	cavica	26		
+Gauchat-1925-480-74	74	to stumble	de t‘“encoubler“	incopulare	26		
+Gauchat-1925-480-75	75	The bridge	Le pont	pŏnte	26	1011	BRIDGE
+Gauchat-1925-480-76	76	is	est en	est ĭn	26		
+Gauchat-1925-480-77	77	stone	pierre	pĕtra	26	857	STONE
+Gauchat-1925-480-78	78	iron	en fer	fĕrru	26	621	IRON
+Gauchat-1925-480-79	79	It is new	Il est neuf	nŏvu	28		
+Gauchat-1925-480-80	80	broad	large	largu	28		BROAD
+Gauchat-1925-480-81	81	The wall	Le mur	mūru	28	933	WALL (OF HOUSE)
+Gauchat-1925-480-82	82	has started	est commencé	cuminitiatu	28		
+Gauchat-1925-480-83	83	The journey	La tour	tŭrre	28	2393	JOURNEY
+Gauchat-1925-480-84	84	(has) started	(est) commencée	cuminitiata	28		
+Gauchat-1925-480-85	85	The journeys	Les tours	tŭrres	30		
+Gauchat-1925-480-86	86	have started	(sont) commencées	cuminitiatas	30		
+Gauchat-1925-480-87	87	The bull	Le bœuf	bŏve	30		BULL
+Gauchat-1925-480-88	88	has horns	a des cornes	habet cŏrn(u)as	30	1393	HORN (ANATOMY)
+Gauchat-1925-480-89	89	The ladder	L‘échelle	scala	30		LADDER
+Gauchat-1925-480-90	90	is leaned	est dressé	directiata	30		
+Gauchat-1925-480-91	91	against	contre	contra	32	2469	AGAINST
+Gauchat-1925-480-92	92	the top	le faîte	fresta	32	1753	TOP
+Gauchat-1925-480-93	93	of the roof	du toit	tēctu	32	769	ROOF
+Gauchat-1925-480-94	94	He closes	Il ferme	claudit	32	1046	SHUT
+Gauchat-1925-480-95	95	the window	la fenêtre	fenĕstra	32	1375	WINDOW
+Gauchat-1925-480-96	96	of the room	de la chambre	pē(n)sile	32	1450	ROOM
+Gauchat-1925-480-97	97	He sweeps	Il balaie	scōpat	34	230	SWEEP
+Gauchat-1925-480-98	98	in front of	devant	de + abante	34	1883	IN FRONT OF
+Gauchat-1925-480-99	99	the door	la porte	pŏrta	34	1567	DOOR
+Gauchat-1925-480-100	100	of the barn	de la grange	granica	34	3025	BARN OR SHED
+Gauchat-1925-480-101	101	We have	On a	hŏmo habet	34		
+Gauchat-1925-480-102	102	lost	perdu	perdūtu	34	1660	LOSE
+Gauchat-1925-480-103	103	the key	la clef	clave	36	1472	KEY
+Gauchat-1925-480-104	104	of the bell tower	du clocher	cloccariu	36		
+Gauchat-1925-480-105	105	It (feminine) is	Elle est	ĭlla est	36		
+Gauchat-1925-480-106	106	lost (feminine)	perdue	perdūta	36		
+Gauchat-1925-480-107	107	Who has	Qui a	cui / qualis habet	36		
+Gauchat-1925-480-108	108	done that?	fait cela?	factu	36		
+Gauchat-1925-480-109	109	I go	Je vais	vado	38		
+Gauchat-1925-480-110	110	to the market	au marché	mercatu	38	633	MARKET
+Gauchat-1925-480-111	111	to search	chercher	circare	38	1468	LOOK FOR
+Gauchat-1925-480-112	112	a shovel	une pelle	pala	38	1901	SHOVEL
+Gauchat-1925-480-113	113	(a) scythe	(une) faux	falce	38	2132	SICKLE OR SCYTHE
+Gauchat-1925-480-114	114	(a) scythe handle	(un) manche de faux	falcariu	38		
+Gauchat-1925-480-115	115	(a) plague	(un) fléau	flagĕllu	40		
+Gauchat-1925-480-116	116	(a) van	(un) van	vannu	40		
+Gauchat-1925-480-117	117	(a) boiler	(une) chaudière	caldaria	40		
+Gauchat-1925-480-118	118	(a) plow	(une) charrue	carrūca	40	1921	PLOUGH
```

Comment:
1921 is the action, I think this should be 2154?

Reply:
Yes, this should be modified.